.github/workflows/trymerge.yml

```diff
@@ -48,6 +48,4 @@ jobs:
             python3 .github/scripts/trymerge.py "${PR_NUM}"
           fi
 
-# TODO: Separate merge on green merges from regular merges to not hold up try-merge workflows overall concurrency
-# NOTE: force pushes are also put in their concurrency group to put them higher than regular merges
-concurrency: try-merge-${{ github.event.client_payload.force}}-${{ github.event.client_payload.on_green }}
+concurrency: try-merge-${{ github.event.client_payload.pr_num }}
```

Comment:
Any new workflows triggered should supercede the old.

Reply:
don't we need cancel-in-progress=true then?